cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,68 @@
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
+  let user;
+  
   before(() => {
-
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
   });
 
-  it('', () => {
-
+  it('should provite the ability to register', () => {
+    cy.get('#firstName')
+      .type(user.firstName);
+    cy.gey('#lastName')
+      .type(user.lastName);
+    cy.get('#userEmail')
+      .type(user.email);
+    cy.log(user.gender);
+    cy.contains('.coustom-control-label', user.gender)
```

Comment:
There's a typo in the CSS class name. It should be '.custom-control-label', not '.coustom-control-label'. Ensure you're using the correct CSS selectors.

Reply:
Done